maint/patches/optional/confdb/sys_lib_dlsearch_path_spec.patch

```diff
@@ -0,0 +1,13 @@
+--- /Users/flyxian/libtool.m4	2019-01-14 16:05:50.000000000 -0600
++++ confdb/libtool.m4	2019-01-14 16:09:25.000000000 -0600
+@@ -2876,7 +2876,9 @@
+   # 'ldconfig -N -X -v | grep ^/' on 64bit Fedora does not report /usr/lib64,
+   # even though it is searched at run-time.  Try to do the best guess by
+   # appending ld.so.conf contents (and includes) to the search path.
+-  if test -f /etc/ld.so.conf; then
++  if test -x /sbin/ldconfig; then
```

Comment:
Should we be checking the version here or something? How do we know this will work vs. the old behavior?

Reply:
I could not find the information about when this problem was fixed. But I came up with a test basically is to see if `ldconfig` is reporting all the path that has been cached in the `ld.so.cache`. I think it is sufficient to determine if the `ldconfig` is good or bad on a given Linux distribution.